pywps/dblog.py

```diff
@@ -170,13 +173,16 @@ def get_session():
                 engine = sqlalchemy.create_engine(database,
                                                   echo=echo,
                                                   connect_args={'check_same_thread': False},
-                                                  poolclass=NullPool)
```

Comment:
Edit: The description below happens in the context of postgres, but I don't think it applies to sqlite... Even then,  the documentation for sqlalchemy mentions: 

> SQLite file-based connections have extremely low overhead, so pooling is not necessary. 

https://docs.sqlalchemy.org/en/13/dialects/sqlite.html#threading-pooling-behavior

I'm pretty sure we want to keep the NullPool here.

- The engine and the connection pool are a global variable. 
- The watchdog process has it in memory, opening connections to query if there are any jobs to do, and keeping these connections open because of this connection pooling.
- When forking happens, a copy of this connection pool is transferred to the forked process, with the same opened connections in the pool.
- When the worker is done processing, the connection pool is deleted and its connections are closed.
- When the watchdog process makes a query to check for any jobs to process, it will use a connection from its pool. If one of these connections had been previously closed, the watchdog will crash.

Reply:
done.